### ISSUE TITLE
AtomicU64 is not a repr(transparent) u64

### ISSUE THREAD

**Issue by Ekleog** [carllerche/loom#51]
Given I think it is not reasonably possible to make it a `repr(transparent)` `u64` without weird behavior arising, maybe just document this fact somewhere prominent, eg. in the README's Limitations section?

**Comment by taiki-e** [comment]
Documented in https://github.com/tokio-rs/loom/pull/253, but is not in the readme or crate-level docs.

**Comment by hawkw** [comment]
Yeah, we should probably put a link in the README/lib.rs docs that links to the docs added in #253.
